### DISCUSSION TITLE
Is random search same as randomized grid search

### DISCUSSION THREAD

**Discussion by ghss-k** [kernc/backtesting.py#747] [Q&A] [answered]
Is random search same as randomized grid search

**Comment by kernc** [comment, marked as answer]
Indeed, [`Backtest.optimize(..., method='grid')`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest.optimize) is randomized grid search trialing over `max_tries=` runs.

**Reply by ghss-k** [reply]
I cant tell which definition is correct

**Reply by kernc** [reply]
Both are. Left is `bt.optimize(method='grid', max_tries=1.)` and right looks like `bt.optimize(method='grid', max_tries=.43)` ...

**Reply by ghss-k** [reply]
Thank you
